modules/comment-likes/admin-style.css

```diff
@@ -0,0 +1,55 @@
+.vers img {
+    display: none;
+}
+
+.metabox-prefs .vers img {
+    display: inline;
+}
+
+.fixed .column-comment_likes {
+    width: 5.5em;
+    padding: 8px 0;
+    text-align: center;
+}
+
+.fixed .column-stats {
+    width: 5em;
+}
+
+.fixed .column-comment_likes .comment-like-count {
+    -webkit-box-sizing: border-box;
```

Comment:
Based on current browser support, I'm pretty sure we don't need to use the `-webkit-` and `-moz-` browser prefixes here.

https://caniuse.com/#feat=css3-boxsizing

cc:  @MichaelArestad